app/models/ems_refresh.rb

```diff
@@ -55,6 +55,19 @@ def self.queue_refresh(target, id = nil, sync = false)
     end
   end
 
+  def self.queue_refresh_new_target(event, ems)
+    MiqQueue.put_or_update(
```

Comment:
This is still going to merge multiple `refresh_new`s together, all you need is `MiqQueue.put`